src/main/java/org/gridsuite/modification/server/dto/LineType.java

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Add a `catalog` directory and move this class and `LineKind` class

Reply:
done